proposals/NNNN-Removing-URL-Param-Max-Length.md

```diff
@@ -0,0 +1,84 @@
+# Removing URL Parameter Max Length
+* Proposal: [SDL-NNNN](NNNN-Removing-URL-Param-Max-Length.md)
+* Author: [Harshavardhan Patankar](https://github.com/hypatankar)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core, HMI, Android, iOS]
```

Comment:
Should `RPC` be included as an impacted platform?

Reply:
Added in commit 5fb96551bc4d6fb18defff370ab83a763dd7eb62.